src/main/java/com/stevedutch/assignment14/web/UserController.java

```diff
@@ -18,7 +18,6 @@ public class UserController {
 
 	@GetMapping("/welcome")
 	public String welcomeUser(ModelMap model) {
-		model.put("user", model);
```

Comment:
same thing as the ChannelController applies here: the welcomeUser method doesn't need to put the whole model in the ModelMap as it's not being used by the frontend